doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -718,6 +718,37 @@ the toplevel, and using them in source files is discouraged.
    the :ref:`command line option <command-line-options>` :n:`-I @string` or by the command :cmd:`Add
    ML Path` `@string` (cf. :cmd:`Declare ML Module`).
 
+.. _extra_dependencies:
+
+Extra Dependencies
+------------------
+
+Dependencies on external files, i.e. non ``.v`` files, can be declared as
+follows:
+
+.. cmd:: From @dirpath Extra Dependency @string {? as @ident }
+   :name: From … Dependency
+
+   Coq checks that a file name :n:`@string` exists in one, and only one,
+   directory associated with :n:`@dirpath`. It does not recursively
+   scan subdirectories but :n:`@string` can contain the directory separator
+   ``/`` and hence denote a file in a subdirectory.
+   The ``coqdep`` tool generates a dependency on the external file for the
+   current ``.v`` file.
+   Note that path elements in :n:`@string` must be valid Coq identifiers,
+   e.g. they cannot contain characters such as ``-`` or ``,``.
+
+When :n:`@ident` is provided, that name can be used by OCaml code, typically
+in a plugin, to access the full path of the external file via the API
+``ComExtraDeps.query_extra_dep``.
+
+This command can also be prefixed with ``Comments``: it has the same effect,
+but the file can be compiled also with version of Coq prior to 8.16.
+
+.. warn:: ``Comments From … Dependency …``
+
+   is deprecated from Coq 8.16 onward.
```

Comment:
```suggestion
This command has been available through the :cmd:`Comments` command,
e.g. :n:`Comments From … Dependency …`.  The :n:`Comments` form is deprecated
in Coq 8.16.
```

IIUC the `.. warn::` is not appropriate because it's not a warning message.  Right?